### DISCUSSION TITLE
Automatically start recording at match start.

### DISCUSSION THREAD

**Discussion by abandonedpools** [IncPlusPlus/csgo-demo-helper#154] [Ideas]
Hi I think it would be a great feature if there was an option to make it automatically record at match start, kinda like [wini's autodemo](https://github.com/iniw/csgo-AutoDemo)

**Comment by IncPlusPlus** [comment]
I'd have to look into whether that'd be possible with the method I'm currently using but that sounds like a great idea.

The only catch is that I'd have to override the prompts about whether the demo is a continuation of a previous match (i.e. you had to reconnect) or whether it's a new match. I suppose this probably won't be an issue though given how the match won't ever start twice in one game. With automatic recording turned on, it would be safe to assume that each match start event would indicate a new match.

Right now my focus is on [my other big project](https://github.com/IncPlusPlus/titanfall2-rp) as well as my job but this is something I definitely _will_ try to do some time in the next month or two. Out of curiosity, how'd you find this project? I never really put it out there so I'm curious how you got here and how you use it.

**Reply by abandonedpools** [reply]
That'd be awesome! It's not necessarily a super important feature or anything but I often forget to start recording at match start and end up getting a cool frag during the game. I use demos to make edits/fragmovie type videos so I much prefer using POV demos rather than GOTV since mouse movement and stuff looks less smoothed out and unnatural. Also to answer your question, my friend sent it to me while we were playing and when I asked where she found it she said that a friend sent it to her, so I don't really know who found it first or how but I guess it's being shared around the community a bit.
